test/py/ganeti.utils.x509_unittest.py

```diff
@@ -34,9 +34,9 @@
 import tempfile
 import unittest
 import shutil
-import OpenSSL
-import distutils.version
 import string
+import OpenSSL
+from packaging import version
```

Comment:
would you mind keep import namespace as it was with distutils: `import packaging.version`? and reference it later as `packaging.version.parse`? Being explicit seems better than implicit.

Reply:
I see no advantage with `import packaging.version` , except longer code.
It would only be better if more than one modules with the name "version" were imported.